apis/Google.Cloud.Diagnostics.AspNetCore/Google.Cloud.Diagnostics.AspNetCore.IntegrationTests/Trace/TraceTest.cs

```diff
@@ -869,12 +910,14 @@ public class PropagationController : Controller
         public static TestServer SecondCallServer;
 
         private IManagedTracer _tracer;
-        private UnchainedTraceHeaderPropagatingHandler _propagatingHandler;
+        private DelegatingHandler _propagatingHandler;
 
-        public PropagationController([FromServices] IManagedTracer tracer, [FromServices] UnchainedTraceHeaderPropagatingHandler propagatingHandler)
+        public PropagationController(
```

Comment:
Unwrap? It looks like the parameter list is reasonably short...

Reply:
Yep, done